dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -688,8 +688,16 @@ public static int call(Logger logger, FuseCallable callable, String methodName,
   public static String getMountedRootPath(AlluxioConfiguration conf, FuseOptions fuseOptions) {
     Optional<UfsFileSystemOptions> options
         = fuseOptions.getFileSystemOptions().getUfsFileSystemOptions();
-    return options.isPresent() ? options.get().getUfsAddress()
-        : conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH);
+    if (conf.getBoolean(PropertyKey.WORKER_FUSE_ENABLED)) {
+      if (!conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH).isEmpty())  {
+        return conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH);
+      }
+    }
+    if (options.isPresent())  {
+      return options.get().getUfsAddress();
+    }
+    // if running at here, return value is an empty String.
+    return conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH);
   }
```

Comment:
Please check it. The old code is that if `getUfsFileSystemOptions()` is successful, then return the Ufs Address as the "MountedRootPath", while we have set the `PropertyKey.FUSE_MOUNT_ALLUXIO_PATH` where we expect alluxio to mount to.